apps/cookbook/src/app/examples/examples.routes.ts

```diff
@@ -483,4 +484,8 @@ export const routes: Routes = [
     path: 'link',
     component: LinkExampleComponent,
   },
+  {
+    path: 'styling-list',
```

Comment:
Should we name this 'Styling HTML Lists' so it is a little more specific when it shows up in the cookbook menu?